ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleManager.java

```diff
@@ -96,6 +96,7 @@ public void onAdEnd(String id, boolean completed, boolean isCTAClicked) {
       public void onAdEnd(String id) {
         if (listener != null) {
           listener.onAdEnd(id);
+          listener.onAdClosed(id);
```

Comment:
this seems to be for fullscreen ads, do we need to add this for fullscreen ads

Reply:
this is for interstitial. don't know why wouldn't they need same callback as others have

Reply:
shouldn't we keep the same order as in https://github.com/Vungle/googleads-mobile-android-mediation/pull/46/files#diff-059c77ef8b7787eb50e9a78d86099aeaL226 ?

Reply:
I think there's no change on fullscreen, and prefer to limit changes only on banner ad.

Reply:
removed